tests/molbe_octane_test.py

```diff
@@ -6,20 +6,12 @@
 from typing import Tuple
 
 import numpy as np
-import pytest
 from pyscf import cc, gto, scf
 
 from quemb.molbe import BE, fragpart
 from quemb.shared.io import write_cube
 
-# TODO: actually add meaningful tests for energies etc.
```

Comment:
Since the test runs now, I would add meaningful tests for energies etc.
(Basically what the TODO says :-D)

This can really be as simple as adding
```
assert np.isclose(mybe.ebe_tot, ...)
assert np.isclose(mybe.ebe_hf, ...)
```
where you get the expected value by simply printing the current value.

Basically we assume it is correct now and want to detect deviations from it.